src/pip/_internal/req/constructors.py

```diff
@@ -74,26 +75,25 @@ def parse_editable(editable_req: str) -> Tuple[Optional[str], str, Set[str]]:
     url_no_extras, extras = _strip_extras(url)
 
     if os.path.isdir(url_no_extras):
-        setup_py = os.path.join(url_no_extras, 'setup.py')
-        setup_cfg = os.path.join(url_no_extras, 'setup.cfg')
+        setup_py = os.path.join(url_no_extras, "setup.py")
+        setup_cfg = os.path.join(url_no_extras, "setup.cfg")
         if not os.path.exists(setup_py) and not os.path.exists(setup_cfg):
             msg = (
                 'File "setup.py" or "setup.cfg" not found. Directory cannot be '
-                'installed in editable mode: {}'
-                .format(os.path.abspath(url_no_extras))
+                "installed in editable mode: {}".format(os.path.abspath(url_no_extras))
```

Comment:
Why is tje first line not touched but the second is LOL

Reply:
Because black uses double quotes everywhere, but for strings with double quotes in the text it uses single quotes.

Personally, I prefer consistency between the two sections of the string, but black disagrees and there's no way to stop it apart from the sledgehammer of `#fmt: off`. But don't get me started on that again 🙂